ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -368,6 +368,20 @@ l2leaf:
         - name: DC1.L2LEAF5B
           id: 15
           mgmt_ip: 192.168.200.121/24
+    # Example for issue 4124
+    - group: DC1_L2LEAF6
+      short_esi: auto # Test Auto ESI to MLAG pair
+      nodes:
+        - name: DC1.L2LEAF6A
+          id: 16
+          mgmt_ip: 192.168.200.122/24
+          uplink_switches: [ DC1-LEAF2A ]
+          uplink_switch_interfaces: [ Ethernet11 ]
```

Comment:
It seems like this collides with some existing server config on LEAF2A/B. Please find an unused interface on those.

Reply:
Thanks, Fixed it.